tsl/src/continuous_aggs/materialize.c

```diff
@@ -421,11 +421,11 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 		if (verbose)
 			elog(INFO,
 				 "new materialization range not found for %s.%s (time column %s): "
-				 "not enough new data past completion threshold (" INT64_FORMAT ")",
+				 "not enough new data past completion threshold as of %s",
```

Comment:
this looks a little strange with integer messages like : not enough new data past completion threshold as of 12. I think the older formatting "not enough new data past completion threshold(12)" looks better.

Reply:
Fixed